lib/misc.py

```diff
@@ -109,5 +109,21 @@ def python_path(cli):
 def reset(cli):
    ch.storage.reset()
 
+def rename(cli):
+   if (not cli.image_ref):
+      ch.FATAL("missing image ref")
+   if (not cli.new_ref):
+      ch.FATAL("missing new image ref")
+   img = ch.Image(ch.Image_Ref(cli.image_ref))
+   new = ch.Image(ch.Image_Ref(cli.new_ref))
+   ch.INFO("rename %s to %s" % (img.ref, new.ref))
+   if (not old.unpack_exist_p):
+      ch.FATAL("image '%s' not in storage" % cli.image_ref)
+   if (new.unpack_exist_p):
+      ch.FATAL("rename: %s already exists in storage" % cli.new_ref)
+   ch.ossafe(os.rename, "can't rename %s" % img.unpack_path, img.unpack_path,
+             new.unpack_path)
```

Comment:
There might be a `ch.rename` already that includes error checking, or possibly I dreamed it?